week-1-practise/level-2/2-format-currency.js

```diff
@@ -6,7 +6,9 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(num) {
+  return "£" + num.toFixed(2)
+}
 
 /* DO NOT EDIT BELOW THIS LINE */
 /* --------------------------- */
```

Comment:
No problems with the code as shown, it meets the needs of this question. You should though bear in mind, that the function is now only usable for calculations that relate to GBP ("£"), as the "£" has been hard-coded into your solution. Have a think about how you might amend the function to make it usable for multiple currencies.